include/sys/thread_stack.h

```diff
@@ -107,7 +107,7 @@ static inline char *Z_THREAD_STACK_BUFFER(k_thread_stack_t *sym)
  * @param size Size of the stack memory region
  */
 #define K_THREAD_STACK_DEFINE(sym, size) \
-	struct z_thread_stack_element __noinit \
+	struct z_thread_stack_element __stackmem \
```

Comment:
think this header file might need an explicit #include <linker/section_tags.h> to pull in the `__stackmem` definition

Reply:
still applies